omnibus/package-scripts/agent/postinst

```diff
@@ -69,8 +69,8 @@ if [ "$DISTRIBUTION" != "Darwin" ]; then
     fi
 
     # Make the system-probe binary and eBPF programs owned by root
-    chown root:root ${INSTALL_DIR}/embedded/bin/system-probe
-    chown -R root:root ${INSTALL_DIR}/embedded/usr/share/system-probe
+    chown root:root ${INSTALL_DIR}/embedded/bin/system-probe || true
+    chown -R root:root ${INSTALL_DIR}/embedded/share/system-probe/ebpf || true
```

Comment:
Why are we okay with this operation failing?

Reply:
We are not ok with it :-) Removed